package.json

```
@@ -53,8 +53,8 @@
     "@swc/plugin-emotion": "^2.5.43",
     "@testing-library/cypress": "^9.0.0",
     "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^12.1.4",
-    "@testing-library/react-hooks": "^7.0.2",
+    "@testing-library/preact": "^3.2.3",
+    "@testing-library/preact-hooks": "^1.1.0",
```

would it help to alias these like you have react?

```suggestion
    "@testing-library/preact": "^3.2.3",
    "@testing-library/preact-hooks": "^1.1.0",
    "@testing-library/react": "npm:@testing-library/preact",
    "@testing-library/react-hooks": "npm:@testing-library/preact-hooks",
```

then the tests can just carry on as if they are using react, as before.

it might help to keep everything feeling like you're using react, even if ultimately you aren't.

---

I suppose it is confusing in tests if it says "@testing-library/preact" rather than "@testing-library-react"! I'll make a change.